dcos-log/journal/reader/config.go

```diff
@@ -52,6 +52,23 @@ func OptionMatch(m []JournalEntryMatch) Option {
 	}
 }
 
+// OptionMatchOR is a functional option that filters entries and applies logical OR to user
+// arguments []JournalEntryMatch.
+func OptionMatchOR(m []JournalEntryMatch) Option {
+	return func(r *Reader) error {
+		if r.Journal == nil {
+			return ErrUninitializedReader
+		}
+
+		for _, match := range m {
+			r.Journal.AddMatch(match.String())
+			r.Journal.AddDisjunction()
```

Comment:
yeah, that's intensional, if i never need to ad logical and to a,b,c it'd be like `a|b|c| a1&b1`
if we don't have the OR at the end, we implicitly say AND (which is wrong)
https://www.freedesktop.org/software/systemd/man/sd_journal_add_match.html